src/tree/localProject/LocalProjectTreeItemBase.ts

```diff
@@ -7,23 +7,26 @@ import * as path from 'path';
 import { AzExtParentTreeItem, TreeItemIconPath } from 'vscode-azureextensionui';
 import { localize } from '../../localize';
 import { treeUtils } from '../../utils/treeUtils';
+import { isMultiRootWorkspace } from '../../utils/workspace';
 
 export abstract class LocalProjectTreeItemBase extends AzExtParentTreeItem {
     public readonly label: string = localize('localProject', 'Local Project');
     public suppressMaskLabel: boolean = true;
     private readonly _projectName: string;
+    private readonly _id: string;
 
     public constructor(parent: AzExtParentTreeItem, projectPath: string) {
         super(parent);
-        this._projectName = path.basename(projectPath);
+        this._projectName = isMultiRootWorkspace() ? `${path.basename(path.dirname(projectPath))}/${path.basename(projectPath)}` : path.basename(projectPath);
```

Comment:
We know the containing WorkspaceFolder when creating the project tree items, so I think it'll simplify things if you pass that into this constructor. For example, I think this would be simpler and more robust:
```typescript
this._projectName = path.relative(path.dirname(folder.uri.fsPath), projectPath);
```